### PR TITLE
Validate View Tests

### PR DESCRIPTION
**Pull request by damonkelley**
Partially completes issue #7

### CODE REVIEW

**Comment by damonkelley** [comment]
ping @somexpert @ldko

File: coda/coda_validate/factories.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+"""Coda MDStore Model factories for test fixtures."""
```
Comment (ldko) [inline coda/coda_validate/factories.py]: wrong app

File: coda/coda_validate/tests/test_views.py [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+        validate = factories.ValidateFactory.create()
+        request = rf.get('/', {'identifier': validate.identifier})
+        response = views.prioritize_json(request)
+        assert response.status_code == 200
+        assert response['Content-Type'] == 'application/json'
+
+    def test_returns_ok_with_invalid_identifier(self, rf):
+        request = rf.get('/', {'identifier': 'dne'})
+        response = views.prioritize_json(request)
+        assert response.status_code == 404
```
Comment (somexpert) [inline coda/coda_validate/tests/test_views.py]: This is testing the the response is a 404, not a 200 "ok".

File: coda/coda_validate/tests/test_views.py [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+
+    def test_template_used(self, client):
+        response = client.get(reverse('coda_validate.views.index'))
+        assert response.template[0].name == 'coda_validate/index.html'
+
+    def test_context(self, client):
+        factories.ValidateFactory.create_batch(30)
+        response = client.get(reverse('coda_validate.views.index'))
+        context = response.context[-1]
+        assert len(context['recently_prioritized']) == 20
```
Comment (ldko) [inline coda/coda_validate/tests/test_views.py]: How does this work that `recently_prioritized` in `context` is always of length 20? If you are generating 30 validate entries and the priority could randomly be 0-9, and the code only takes those with priority greater than 0 before cutting to 20...I am confused.

Comment (damonkelley) [inline coda/coda_validate/tests/test_views.py]: I am not sure I follow. `recently_prioritized` is just the first 20 results of a query to get all Validate objects with a priority > 0 in reverse order by the change date.

Comment (ldko) [inline coda/coda_validate/tests/test_views.py]: What if there are only 10 entries that have priority > 0? If you didn't specify when generating the batch of validates, isn't it just random (maybe not that random by how FuzzyInteger works?) chance that it ended up with a batch with >= 20 `priority` > 0?

Comment (damonkelley) [inline coda/coda_validate/tests/test_views.py]: absolutely

Comment (damonkelley) [inline coda/coda_validate/tests/test_views.py]: modified in ed03c2d

**Comment by ldko** [comment]
:+1:

**Comment by somexpert** [comment]
:thumbsup:

**Pull request merged**